src/Backend/Modules/MiniBlog/Installer/Installer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Backend\Modules\MiniBlog\Installer;
+
+use Backend\Core\Installer\ModuleInstaller;
+
+/**
+ * This action will install the mini blog module.
+ *
+ * @author Lander Vanderstraeten <lander.vanderstraeten@wijs.be>
+ */
+class Installer extends ModuleInstaller
+{
+    /**
+     * Install the module
+     */
+    public function install()
+    {
+        $this->importSQL(dirname(__FILE__) . '/Data/install.sql');
+
+        $this->addModule('MiniBlog');
+
+        $this->importLocale(dirname(__FILE__) . '/Data/locale.xml');
+
+        $this->makeSearchable('MiniBlog');
```

Comment:
After the ->addModule call, you can use $this->getModule() everywhere instead of the string 'miniblog'. This makes your code easier to reuse in other modules.